bot/controllers/userAdd.js

```diff
@@ -0,0 +1,28 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+class UserAdd extends BaseController {
+  constructor(message) {
+    super(message);
+
+    const controller = this;
+
+    this.commands = [
+      new Command(
+        '!resend',
+        '!resend',
+        'Resend Message',
```

Comment:
The context that you are adding to the !help command is also ambiguous. Resend what message? Consider that this is a messaging platform where users will be regularly sending many messages to one another, each channel, and directly to the bots.